### DISCUSSION TITLE
Recommended nRF52840 RCP implementation?

### DISCUSSION THREAD

**Discussion by fgervais** [openthread/openthread#8779] [Q&A] [answered]
I see that there is two implementations:

- https://github.com/openthread/ot-nrf528xx
- https://github.com/nrfconnect/sdk-nrf/tree/main/samples/openthread/coprocessor

Which one would be recommended for the development of a new product?

**Comment by jwhui** [comment, marked as answer]
For new product, Nordic recommends the [nRF Connect SDK](https://www.nordicsemi.com/Products/Development-software/nrf-connect-sdk).

**Reply by fgervais** [reply]
If it's the case, in my opinion something should be done to make it clear.

At least for me the repo hosted on `openthread` looks more official than something in a `sample` folder. Even the new `Note:` (https://github.com/openthread/ot-nrf528xx/commit/8c6658f6263a8ec1e8135534133a96770b9bb065) does not really make sense as when we use the `ot-nrf528xx` repo we are not really aware of which backend we use.

**Reply by jwhui** [reply]
We tried to make this clear with:
- Title: "OpenThread on nRF52840 **Example**"
- First line: "This directory contains **example** platform drivers for Nordic Semiconductor nRF52840 SoC"
- First callout: "Note: For new projects, you should use nRF Connect SDK."
- Sub title: "Building the **examples**"
